auction-app-backend/src/main/java/com/example/auctionapp/controller/TestController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.auctionapp.controller;
+
+import com.example.auctionapp.entity.TestEntity;
+import com.example.auctionapp.repository.TestRepository;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/test")
+public class TestController {
+
+    private final TestRepository testRepository;
+
+    public TestController(TestRepository testRepository) {
+        this.testRepository = testRepository;
+    }
+
+    @GetMapping("/")
```

Comment:
If we don't add anything to the url we have set in RequestMapping, there is no need for `"/"`. 
Here @GetMapping works fine.